src/minuteman_nfq.erl

```diff
@@ -84,6 +84,7 @@ start_link(Queue) ->
   {ok, State :: #state{}} | {ok, State :: #state{}, timeout() | hibernate} |
   {stop, Reason :: term()} | ignore).
 init([Queue]) ->
+  process_flag(trap_exit, true),
```

Comment:
This is a fix, but fundamentally when minuteman_nfq processes die, the ports linked to them (and therefore the FDs) should be shutdown / closed. Can we see if that's possible? I can dig into this a little bit more.